app.js

```diff
@@ -1,14 +1,20 @@
 const express = require("express");
 const app = express();
-const { getTopics } = require("./controllers/topic.controller")
+const { getTopics, getArticle } = require("./controllers/controller-connection")
 
 app.use(express.json());
 
 app.get("/api/topics", getTopics);
 
+app.get("/api/articles/:article_id", getArticle)
+
 app.use((req, res, next) => {
     res.status(404).send({ msg: 'path not found' })
 })
 
+app.use((err, req, res, next) => {
+    res.status(err.status).send({ msg: err.msg })
```

Comment:
This error handler works in the case you're testing for but I'd recommend putting a 500 error handler in at this stage to catch any unexpected errors in the future.